sdk/formrecognizer/ai-form-recognizer/CHANGELOG.md

```diff
@@ -3,6 +3,30 @@
 ## 1.0.0-preview.3 (Unreleased)
 
 - Blank pages in receipt recognition are now handled properly.
+- Support Azure Active Directory credential.
+- Support to copy a custom model from one Form Recognizer resource to another.
+- Refactoring for cross-language consistency:
+  - [Breaking] Rename `beginRecognizeForms()` to `beginRecognizeCustomForms()` in `FormRecognizerClient`.
+  - [Breaking] Rename `listModels()` to `listCustomModels()` in `FormTrainingClient`.
+  - [Breaking] Rename `count` to `customModelCount` and `limit` to `customModelLimit` in `AccountProperties`.
+  - [Breaking] Rename type `ErrorInformation` to `FormRecognizerError`.
+  - [Breaking] Rename type `ModelStatus` to `CustomFormModelStatus`.
+  - [Breaking] Rename type `CustomFormSubModelField` to `CustomFormField`.
+  - [Breaking] Rename type `FormElement` to `FormContent` and `FormElementCommon` to `FormContentCommon`.
+  - [Breaking] Rename property `fieldLabel` to `labelText` in `FormField` type.
+  - [Breaking] Rename type `ModelInfo` to `CustomFormModelInfo`.
+  - [Breaking] Rename tppe `TrainStatus` to `TrainingStatus`.
+  - [Breaking] Rename `models` to `submodels` in `CustomFormModel`.
+  - [Breaking] Recognition methods and training methods now return the result directly, instead of wrapping them in a response object. Specifically,
+    - `beginTraining` now returns `CustomFormModel` instead of `FormModelResponse` from the poller.
+    - `beginRecognizeContent` and `beginRecognizeContentFromUrl` now return `FormPageArray` instead of `RecognizeContentResultResponse` from the poller.
+    - `beginRecognizeForms` and `beginRecognizeFormsFromUrl` now return `RecognizedFormArray` instead of `RecognizeFormResultResponse` from the poller.
+    - `beginRecognizeReceipts` and `beginRecognizeReceiptsFromUrl` now return `RecognizedReceiptArray` instead of `RecognizeReceiptResultResponse` from the poller.
+  - [Breaking] Remove `getFormTrainingClient()` from `FormRecognizerClient`. A new method `getFormRecognizerClient()` is added to `FormTrainingClient`
+  - Rename the first parameter of `beginRecognizeContentFromUrl()` from `documentUrl` to `formUrl`.
```

Comment:
I see you don't have this in master yet, but just in case I wanted to mention that we're also changing the parameter in `beginRecognizeReceipts` to `receipt`, and `beginRecognizeContent`,`beginRecognizeCustomForms` to `form`

Reply:
Thanks! I think I'd like to keep the parameter name `data` for JS

Reply:
I'm not sure if we've talked about this before, but is there a reason why JS would diverge here

Reply:
Not strong opinion, this is input and I don't like to pre-classify it to "form" or "receipt" since users could pass in any binary data. Probably also the reason why I had "documentUrl" instead of "formUrl" and "receiptUrl". 

Has "formData"/"receiptData" been considered?

Reply:
Anyway, I submitted PR https://github.com/Azure/azure-sdk-for-js/pull/9157 to use "form" and "receipt"